test/expected/plan_ordered_append-10.out

```diff
@@ -764,10 +764,10 @@ LIMIT 2;
                ->  Index Scan using _hyper_3_7_chunk_dimension_last_time_idx on _hyper_3_7_chunk
          ->  Materialize
                ->  Append
-                     ->  Seq Scan on _hyper_4_11_chunk
-                     ->  Seq Scan on _hyper_4_12_chunk
-                     ->  Seq Scan on _hyper_4_13_chunk
                      ->  Seq Scan on _hyper_4_14_chunk
+                     ->  Seq Scan on _hyper_4_13_chunk
+                     ->  Seq Scan on _hyper_4_12_chunk
+                     ->  Seq Scan on _hyper_4_11_chunk
 (15 rows)
 
 -- test INNER JOIN against non-hypertable
```

Comment:
maybe add another tests `ORDER BY dimension_last.time ASC` to double check the order changes?